charts/nri-prometheus/values.yaml

```diff
@@ -220,41 +194,51 @@ config:
   #         - namespace
   #         - node
 
-# If you wish to provide additional annotations to apply to the pod(s), specify
-# them here
-# podAnnotations:
-
-# If you wish to provide a pod security context, specify it here
-# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
-# Kind: PodSecurityContext
-# podSecurityContext:
-
-# Pod scheduling priority
-# Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
-# priorityClassName: high-priority
-
+# -- Annotations to be added to all pods created by the integration.
+podAnnotations: {}
+
+
+# These values are managed by the common-library
```

Comment:
also for this chart I do not see much value into creating a different section for this

Reply:
I applied here the suggestions from newrelic/nri-kube-events#112

Reply:
Solved in [#4c27761](https://github.com/newrelic/nri-prometheus/pull/289/commits/4c27761bd01799514b1cb1a35cf6d70f77565200) and [#cc2ab29](https://github.com/newrelic/nri-prometheus/pull/289/commits/cc2ab298fbfd49ff712d8351be912afbed4685dd)